source/Sashimi.Tests/ActionHandlersFixture.cs

```diff
@@ -672,12 +668,11 @@ public void InlineHclTemplateWithMultilineOutput()
                                                                         .Be($"{expected.Replace("\r\n", "\n")}");
                                                                    });
         }
-        
+
         [Test]
         public void InlineJsonTemplateAndVariables()
         {
             IgnoreIfVersionIsNotInRange("0.11.15", "0.15.0");
```

Comment:
Do we need to change these @geofflamrock ?

Reply:
Hmmm yeah good thought, we probably should increase to include 1.0 as they are valid for that.

Reply:
I'm thinking we might be able to just remove the max version here instead of having to bump it all the time, can you think of any reason against doing that @johnsimons ?

Reply:
We did some more digging and changes don't need to be made here as it is testing parameters that used to be a thing in <0.15 and no longer apply.